lib/retriable.rb

```diff
@@ -37,6 +37,7 @@ def retriable(opts = {})
     on_retry          = local_config.on_retry
     sleep_disabled    = local_config.sleep_disabled
 
+    exception_list = on.is_a?(Hash) ? on.keys : on
```

Comment:
(i moved this because i felt like it was hanging out in an awkward place way underneath all the other instantiation variables)